package.json

```diff
@@ -72,7 +72,7 @@
     "prettier": "^1.18.2",
     "standard-version": "^7.0.0",
     "uglify-js": "^3.6.0",
-    "webpack": "^4.38.0"
+    "webpack": "^4.47.0"
```

Comment:
we dont do reverse updates. If there's a problem with nodejs v18 you can install an older version of terser instead.